docs/design/bundle-sources.md

```diff
@@ -0,0 +1,240 @@
+# Building a file-based catalog from a plain bundle image
+
+**IMPORTANT:** Operator Lifecycle Manager (OLM) v1 features and components are still experimental. Early adopters and contributors should expect breaking changes. The following procedures are not recommended for use on production clusters.
+
+You can build a static collection of arbitrary Kubernetes manifests in the YAML format, or *plain bundle*, and add the image to a file-based catalog (FBC). The experimental `olm.bundle.mediatype` property of the `olm.bundle` schema object differentiates a plain bundle from a regular bundle. You must set the bundle media type property to `plain+v0` to specify a plain bundle.
+
+For more information, see the [Plain Bundle Specification](https://github.com/operator-framework/rukpak/blob/main/docs/bundles/plain.md) in the RukPak repository.
+
+To build a file-based catalog from a plain bundle image, you must complete the following steps:
+
+* Create a plain bundle image
+* Create a file-based catalog
+* Add the plain bundle image to your file-based catalog
+* Build your catalog as an image
+* Publish your catalog image
+
+## Building a plain bundle image from an image source
+
+Currently, the Operator Controller only supports building a plain bundle image from an image source.
+
+*Prerequisites*
```

Comment:
Nit: If this is a section title make it a markdown header instead of italicized. Markdown headers are linkable which makes it easier to share and link to sections within the document. In this case I would do something like:
```suggestion
### Prerequisites
```

This suggestion goes for all subheadings that are italicized.

Reply:
@rashmigottipati Try the following to make a unique header for the prereqs and procedures:
```suggestion
<h3 id="prereqs-building-plain-bundle-from-image">
Prerequisites
</h3>
```